data/winevt_features.yaml

```diff
@@ -648,3 +688,283 @@ system_7045_v0:
     - name: account_name
       string_index: 4
 
+# Microsoft-Windows-TerminalServices-LocalSessionManager/Operational
```

Comment:
Do you happen to have a reference for the `Microsoft-Windows-TerminalServices-LocalSessionManager` events?